README.md

```diff
@@ -39,6 +47,10 @@ eslint-find-new-rules ./index.js
 
 This is resolved relative to the `process.cwd()` which, in the context of npm scripts is always the location of your `package.json`.
 
+#### Supported file types
+
+You may specify any [config file originally supported by ESLint](http://eslint.org/docs/user-guide/configuring) (`.eslintrc.*` or a `package.json` containing an `eslintConfig` field) as well as any JavaScript, JSON or YAML file containing a proper eslint configuration.
```

Comment:
I think it's clear enough to simply state: `You may specify any [config format supported by ESLint](http://eslint.org/docs/user-guide/configuring)`